### DISCUSSION TITLE
What should we do with the default `core_components` file from the Phoenix 1.7

### DISCUSSION THREAD

**Discussion by andyduong1920** [nimblehq/elixir-templates#322] [RFCs]
As you might be aware, Phoenix 1.7 will generate the CoreComponents file, which contains most of the components like `input, form, button, ...` using Tailwindcss.

Most of our project requires custom CSS. So I'm asking your opinion about when the user selected the TailwindCSS addon.

Btw, if the user selects the Bootstrap addon, we can remove the CoreComponent as it's useless

**Comment by hanam1ni** [comment]
I voted "Tailwindcss addon selected - Option 2" 🙌 IMO, we should keep the CoreComponents module as the template may be used by various types of applications. Thus we should keep this in case some projects such as the admin panel, or back-office app, won't need much custom styling or branding.

**Reply by andyduong1920** [reply]
Yeah make sense @hanam1ni 👍

**Comment by andyduong1920** [comment]
I realized the Vote doesn't allow multiple selections, so I updated the vote content a bit FYI
